docs/state/var-operations.md

```diff
@@ -0,0 +1,529 @@
+    var_1: bool = True
+    var_2: bool = True
+
+    def update(self):
+        self.var_1 = random.choice([True, False])
+        self.var_2 = random.choice([True, False])
+
+def var_logical_example():
+    return rx.vstack(
+        rx.table_container(
+            rx.table(
+                headers=["Var 1", "Var 2", "Operation", "Outcome"],
+                rows=[
+                    (f"{VarLogicalState.var_1}", f"{VarLogicalState.var_2}", "Logical AND (&)", f"{VarLogicalState.var_1 & VarLogicalState.var_2}"),
+                    (f"{VarLogicalState.var_1}", f"{VarLogicalState.var_2}", "Logical OR (|)", f"{VarLogicalState.var_1 | VarLogicalState.var_2}"),
+                    (f"{VarLogicalState.var_1}", f"{VarLogicalState.var_2}", "The invert of Var 1 (~)", f"{~VarLogicalState.var_1}"),
+                    ],
+                variant="striped",
+                color_scheme="green",
+            ),
+        ),
+        rx.button("Update", on_click=VarLogicalState.update),
+    )
+```
+
+```python eval
+docdemo_from(VarLogicalState, component=var_logical_example, imports=["import random"])
+```
+
+### Contains, Reverse and Join
+
+
+The 'in' operator is not supported for Var types, we must use the `Var.contains()` instead. When we use `contains`, the var must be of type: `dict`, `list`, `tuple` or `str`. 
+`contains` checks if a var contains the object that we pass to it as an argument.
+
+We use the `reverse` operation to reverse a list var. The var must be of type `list`.
+
+Finally we use the `join` operation to join a list var into a string. 
+
+```python exec
+class VarListState(State):
+    list_1: list = [1, 2, 3, 4, 6]
+    list_2: list = [7, 8, 9, 10]
+    list_3: list = ["p","y","t","h","o","n"]
+
+def var_list_example():
+    return rx.vstack(
+        rx.heading(f"List 1: {VarListState.list_1}", size="lg"),
+        rx.heading(f"List 2: {VarListState.list_2}", size="lg"),
+        rx.heading(f"List 3: {VarListState.list_3}", size="lg"),
+        
+        rx.heading(f"Does List 1 contain the number 3: {VarListState.list_1.contains(3)}", size="md"),
+        rx.heading(f"Reverse List 2: {VarListState.list_2.reverse()}", size="md"),
+        rx.heading(f"Join List 3 into string: {VarListState.list_3.join()}", size="md"),
+
+    )
+```
+
+```python eval
+docdemo_from(VarListState, component=var_list_example)
+```
+
+
+
+### Lower, Upper, Split
+
+The `lower` operator converts a string var to lowercase. The `upper` operator converts a string var to uppercase. The `split` operator splits a string var into a list.
+
+```python exec
+class VarStringState(State):
+    string_1: str = "PYTHON is FUN"
+    string_2: str = "react is hard"
+   
+
+def var_string_example():
+    return rx.vstack(
+        rx.heading(f"List 1: {VarStringState.string_1}", size="lg"),
+        rx.heading(f"List 2: {VarStringState.string_2}", size="lg"),
+
+        rx.heading(f"Lower Case output of List 1: {VarStringState.string_1.lower()}", size="md"),
```

Comment:
Same comment about how to strucutre this section with hstack and vstacks as Contains, Reverse and Join section,

Can shorten the comments up to 

List 1 Lower Case: python is fun
List 2 Upper Case: REACT IS HARD

Reply:
done